core/include/detray/core/detector.hpp

```diff
@@ -255,11 +255,15 @@ class detector {
         -> const detector_volume<detector> {
         // The 3D cylindrical volume search grid is concentric
         const transform3 identity{};
+
         const auto loc_pos =
             _volume_finder.global_to_local(identity, p, identity.translation());
 
+        printf("%f %f %f \n", loc_pos[0], loc_pos[1], loc_pos[2]);
+
         // Only one entry per bin
         dindex volume_index{*_volume_finder.search(loc_pos)};
```

Comment:
This line yields a seg fault in `tools_propagator.cpp` when `loc_pos` is `(33.150810, -3.127880, 455.621918)`. @niermann999 any idea why it happens?

Reply:
The volume grid is not yet filled or tested for the toy detector, which is why the ```volume_by_pos``` call is not yet used in the navigator and why we cannot start the navigation outside of volume 0. This simply was not high on my list before, sorry

What I am surprised about is that the segfaults in the CI happen in the straight line navigation, where overstepping should not be possible...

Reply:
Hmm maybe I should have not commented to remove the volume_bounds in the description haha